### PR TITLE
Add kubectl-minio plugin

### PR DESCRIPTION
**Pull request by nitisht**

### CODE REVIEW

**Comment by nitisht** [comment]
@ahmetb @corneliusweig this is now ready for review. PTAL.

File: plugins/minio.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      specify -n/--namespace explicitly (the current namespace setting is not
+      yet used).
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/minio/operator/releases/download/v3.0.19/kubectl-minio_darwin_amd64.zip
+    sha256: d5aaef0d5c30a3e0532c0729ac0f8bb7a4f966de062e045ab5fd73fcedaa8aaf
+    files:
```
Comment (ahmetb) [inline plugins/minio.yaml:23]: Files: is optional, since you got the default value here, you can delete it

File: plugins/minio.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: minio
+spec:
+  version: v3.0.19
+  homepage: https://min.io
+  shortDescription: Deploy and manage MinIO Operator and MinIO Tenant
```
Comment (ahmetb) [inline plugins/minio.yaml:8]: Try to shorten a bit if you can. Maybe don't repeat minio second time

File: plugins/minio.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: minio
+spec:
+  version: v3.0.19
+  homepage: https://min.io
```
Comment (ahmetb) [inline plugins/minio.yaml:7]: If there's a specific page talking about the kubectl plugin, please link to it.

**Review by ahmetb — APPROVED** [review]
Mostly looks good. 
/approce

**Comment by nitisht** [comment]
Thanks for the review @ahmetb . Addressed the comments.

**Pull request merged**